packages/deploy-utils/src/setup-credentials/index.ts

```diff
@@ -26,60 +28,16 @@ const sharedKeys = {
 const base: Record<TargetEnvironment, Record<string, string | undefined>> = {
   staging: {
     ...sharedKeys,
-    REACT_APP_API_URL: "https://evergreen-staging.corp.mongodb.com/api",
-    REACT_APP_EVERGREEN_URL: "https://evergreen-staging.corp.mongodb.com",
-
-    // TODO: Deduplicate Spruce and Parsley vars
-    REACT_APP_GRAPHQL_URL:
-      "https://evergreen-staging.corp.mongodb.com/graphql/query",
-    REACT_APP_GQL_URL:
-      "https://evergreen-staging.corp.mongodb.com/graphql/query",
-
-    REACT_APP_HONEYCOMB_BASE_URL:
-      "https://ui.honeycomb.io/mongodb-4b/environments/staging",
     REACT_APP_HONEYCOMB_ENDPOINT: process.env.HONEYCOMB_ENDPOINT_STAGING,
     REACT_APP_HONEYCOMB_INGEST_KEY: process.env.HONEYCOMB_INGEST_KEY_STAGING,
```

Comment:
What if these variables were added to a task's environment via a function variable passed to the `env` param? Then we could specify the value for staging vs beta vs prod tasks, and potentially delete this file altogether 👀